rep-I0005.rst

```diff
@@ -0,0 +1,164 @@
+::
+
+   REP: I0005
+   Title: Cartesian Motion Extensions for the Simple Message Protocol
+   Authord: Fred Proctor <proctor@nist.gov> and Tre Harrison <william.harrison@nist.gov>
+   Status: Active
+   Type: Process
+   Content-Type: text/x-rst
+   Created: 20-May-2015
+   Post-History: 20-May-2015
```

Comment:
Officially, this field is used when the REP has been posted to the `ros-users` mailing list, or in our case probably `ros-industrial` (`swri-ros-pkg-dev`).